buf.yaml

```diff
@@ -0,0 +1,10 @@
+version: v2
+modules:
+  - path: internal/proto
+deps:
+  - buf.build/bufbuild/protovalidate
+lint:
+  except:
+    - FIELD_NOT_REQUIRED
```

Comment:
This is a new lint rule - kept it after migration since it failed on one of the test proto files.